node/src/effect.rs

```diff
@@ -136,17 +137,30 @@ pub type Effects<Ev> = Multiple<Effect<Ev>>;
 /// size of two items is chosen because one item is the most common use case, and large items are
 /// typically boxed. In the latter case two pointers and one enum variant discriminator is almost
 /// the same size as an empty vec, which is two pointers.
-type Multiple<T> = SmallVec<[T; 2]>;
+// TODO: Move this to a more central spot.
+pub(crate) type Multiple<T> = SmallVec<[T; 2]>;
```

Comment:
Let's keep this private, and duplicate the alias in the new test code if you really think it's worthwhile over a `Vec`.

Reply:
The fact that this _wasn't_ public before is just by accident -- `SmallVec` itself is put and the type alias was not required to be public for this reason.

Many public functions of the effect builder take `Multiple` as an argument. There is no convenient way to specify these types without making this public.

We can agree to keep it inside `effect.rs` though, I'll remove the comment.